match/spec/commands_generator_spec.rb

```diff
@@ -89,6 +89,7 @@ def expect_githelper_clone_with(git_url, shallow_clone, git_branch)
     end
 
     it "can use the git_url short flag from tool options" do
+      stub_const('ENV', {'MATCH_PASSWORD' => ''})
```

Comment:
@joshdholtz could you tell why this was error'ing in CI and not locally in my machine when running `bundle exec rspec`? And how did you come to this fix? 🤯 😅

Reply:
@rogerluan CircleCI gets really weird sometimes and just needs a restart 🤷‍♂️